src/index.ts

```diff
@@ -1,5 +1,21 @@
-import { Generator } from "./Generator";
-import { VersionManager } from "./VersionManager";
-import { TitaniaRelic, TitaniaRelicLocation, TitaniaRelicReward, TitaniaRelicRewardItem, TitaniaRelicVaultedInfo, TitaniaWFMInfo} from "./Types"
+import Generator from './Generator';
+import VersionManager from './VersionManager';
+import {
+  TitaniaRelic,
+  TitaniaRelicLocation,
+  TitaniaRelicReward,
+  TitaniaRelicRewardItem,
+  TitaniaRelicVaultedInfo,
+  TitaniaWFMInfo,
+} from './Types';
 
-export { Generator, VersionManager, TitaniaRelic, TitaniaRelicLocation, TitaniaRelicReward, TitaniaRelicRewardItem, TitaniaRelicVaultedInfo, TitaniaWFMInfo };
\ No newline at end of file
+export {
+  Generator,
+  VersionManager,
+  TitaniaRelic,
+  TitaniaRelicLocation,
+  TitaniaRelicReward,
+  TitaniaRelicRewardItem,
+  TitaniaRelicVaultedInfo,
+  TitaniaWFMInfo,
+};
```

Comment:
If all files would follow the same pattern in `Config.ts` you could do something like this:
```ts
export * from './Generator';
export * from './VersionManager';
export * from './Types';
```

Reply:
if you test it and it works, lemme know. i'm working right now so i can't spin it up to test the types. i was having issues resolving types from a cjs project before to get the types properly, so at worst i would be stuck with the same non-working types

Reply:
> i think that depends on your ts config

It's not a ts config thing, lemme explain.
export * from '...' transpiles to this:
```js
_exportStar(require("..."), exports);
function _exportStar(from, to) {
    Object.keys(from).forEach(function(k) {
        if (k !== "default" && !Object.prototype.hasOwnProperty.call(to, k)) Object.defineProperty(to, k, {
            enumerable: true,
            get: function() {
                return from[k];
            }
        });
    });
    return from;
}
```
But for this to work all modules need to be in the same format, just to remind:
```ts
export const Config = {....

export default Config;
```

Reply:
I can refactor everything to be like this for you, but i need to know if you and @Soundofdarkness agree to it.

Reply:
that's gross

Reply:
yay, another piece of secret underlying magic that i hate about ts

Reply:
i'm fine with it, but i'd rather wait for @Soundofdarkness

Reply:
> yay, another piece of secret underlying magic that i hate about ts

But it's an ecmascript thing. Not a typescript one, i just thought it you would like code instead of [specificaiton](https://262.ecma-international.org/#exportentry-record).

Reply:
i don't like ecma magic. i'd prefer exporting a file and it exporting things from it, not "..."

Reply:
> i don't like ecma magic. i'd prefer exporting a file and it exporting things from it, not "..."

Sorry for not clarifying the '...' is just a placeholder for the module path.

Reply:
ah, ok, that's fine